market/connector/google-geo-api/README.md

```diff
@@ -0,0 +1,8 @@
+Axon Ivy's [Google Map API] (https://developers.google.com/maps/documentation/javascript/overview) lets you customize maps with your own content and imagery for display on web pages and mobile devices. The Google Map API features four basic map types (roadmap, satellite, hybrid, and terrain) which you can modify using layers and styles, controls and events, and various services and libraries.
```

Comment:
Can you check the README.md @andreasbalsiger  and give feedback?

Reply:
Axon Ivy's [Google Maps API] (https://developers.google.com/maps/documentation/javascript/overview) connector lets you customize maps with your own content and imagery for display on web pages and mobile devices. The connector:

- Features four basic map types (roadmap, satellite, hybrid, and terrain) which you can modify using layers and styles, controls and events, and various services and libraries.
- Provides access to the core API of Google Maps Platform
- Supports you with an easy-to-copy demo implementation to reduce your integration effort.
- Enables low code citizen developers to integrate common API's without writing a single line of code.